parser-typechecker/src/Unison/Result.hs

```diff
@@ -30,7 +27,6 @@ data Note v loc
   | TypeError (Context.ErrorNote v loc)
   | TypeInfo (Context.InfoNote v loc)
   | CompilerBug (CompilerBug v loc)
-  deriving (Show)
```

Comment:
Hrmm, does this have to be removed? Show instances are handy for debugging.

Reply:
No; adding it (and others) back. I had initially removed them all only to audit all call sites to all of the `Show` instances that were transitively depending on `Name`'s show, to switch them over to calling `toText` instead. They could/should be added back (although I will note that `recover-rtti` seems a lot better for debugging to me - no need to muscle through `Show` instances everywhere, including on types that can't have a `Show` instance for whatever reason)